gnomad/utils/annotations.py

```diff
@@ -2794,7 +2794,7 @@ def get_copy_state_by_sex(
     :param locus_expr: LocusExpression of the variant.
     :param is_xx_expr: BooleanExpression indicating whether the sample has an XX sex
        karyotype.
-    :return: A tuple of BooleanExpressions:
+    :return: Tuple of BooleanExpressions:
```

Comment:
it just occurred to me that this is assuming the sample only contains XX and XY karyotypes (i.e., no samples have ambiguous sex or aneuploidies). can you add that into the docstring?